src/mpid/ch4/src/ch4_probe.h

```diff
@@ -31,6 +32,7 @@ MPL_STATIC_INLINE_PREFIX int MPID_Probe(int source,
         goto fn_exit;
     }
 
+    av = MPIDIU_comm_rank_to_av(comm, source);
```

Comment:
The code in the correspoding netmod functions is using `MPIDI_OFI_av_to_phys` (no translation, just read av) instead of the old `MPIDI_OFI_comm_to_phys` (has translation). So I think the translation to AV should happen here and both cases (execlusive/non-execlusive SHM) would only use the translated AV for communication.